browser.js

```diff
@@ -1,16 +1,27 @@
 'use strict'
 
+var format = require('quick-format-unescaped')
+
 module.exports = pino
 
 var _console = global.console || {}
-function Pino () {}
-Pino.prototype = _console
 
 function pino (opts) {
   opts = opts || {}
+  opts.browser = opts.browser || {}
+  var proto = opts.browser.write || _console
+  if (opts.browser.write) opts.browser.asObject = true
+
+  var levels = ['error', 'fatal', 'warn', 'info', 'debug', 'trace']
```

Comment:
I guess you can move that array into the `asObject` function. You don't need it anywhere else.